packages/devtools_app/lib/src/service/service_manager.dart

```diff
@@ -212,10 +212,17 @@ class ServiceConnectionManager {
 
   // TODO(kenz): consider caching this value for the duration of the VM service
   // connection.
+  /// Returns the root package directory for the main isolate.
+  ///
+  /// If a non-null value is returned, the value will be a file URI String and
+  /// it will NOT have a trailing slash.
```

Comment:
Is it worth adding asserts for this so if it happens during tests it would throw?